.travis.yml

```diff
@@ -94,14 +107,14 @@ after_script:
       pip install -U codecov
       codecov -e TEST_SUITE
     fi
-  - ./bin/yarn global add @zeus-ci/cli
-  - $(./bin/yarn global bin)/zeus upload -t "text/xml+xunit" .artifacts/*junit.xml
-  - $(./bin/yarn global bin)/zeus upload -t "text/xml+coverage" .artifacts/*coverage.xml
-  - $(./bin/yarn global bin)/zeus upload -t "text/xml+coverage" .artifacts/coverage/cobertura-coverage.xml
-  - $(./bin/yarn global bin)/zeus upload -t "text/html+pytest" .artifacts/*pytest.html
-  - $(./bin/yarn global bin)/zeus upload -t "text/plain+pycodestyle" .artifacts/*pycodestyle.log
-  - $(./bin/yarn global bin)/zeus upload -t "text/xml+checkstyle" .artifacts/*checkstyle.xml
-  - $(./bin/yarn global bin)/zeus upload -t "application/webpack-stats+json" .artifacts/*webpack-stats.json
+  - yarn global add @zeus-ci/cli
+  - $(yarn global bin)/zeus upload -t "text/xml+xunit" .artifacts/*junit.xml
```

Comment:
Follow up idea: might be easier to do `PATH="$(yarn global bin):${PATH}"` once and then simply running `zeus`.